dfstools/src/csv_tools.py

```diff
@@ -32,7 +32,11 @@ def load_csv_to_df(path, include_hidden=False, traverse_subdir=True, follow_syml
     ###
 
     # mock-up for demonstration - remove after development
-    dataframe_list = ['airlines', 'airports', 'flights', 'trip_logs']
-
+    #dataframe_list = ['airlines', 'airports', 'flights', 'trip_logs']
+    dataframe_dict = {'airlines': pd.read_csv('data/airlines/airlines.csv'),
```

Comment:
Keep in mind this will only work on a Mac, on a PC the path names are data\\airlines\\airlines.csv and so-forth

Reply:
I retract my statement - apparently pandas.read_csv() does work out the path so that it works cross-platform automatically.  None the less I'll be committing some example code that shows a [better way](https://github.com/NCATComp410/comp410_summer2020/blob/5d557ef4660469730d43addb66fd82e6be5ed5e4/demo.py#L40) to handle this.